src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="header__logo"
+            class="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div
+        class="container"
+        data-qa="card"
+      >
+        <img
+          class="container__image"
+          src="images/imac.jpeg"
+          alt="Apple Device(imac)"
+        />
+        <div class="container__product-info">
+          <h3 class="product-info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="product-info-text">Product code: 195434</p>
+        </div>
+        <div class="container__stars-reviews">
+          <div class="stars stars--4">
```

Comment:
BEM modifier 'stars--4' suggests that there are four stars, but there are five star elements. Ensure that the modifier correctly reflects the number of active stars.